notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<p>I am not sure if we can make the postprocess part of each meter reader configurable instead of hard code. For example, a mapping list for these parameters ? so some non-developer person can help to modify it according their meters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1030/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>